packages/core/test/toast/overlayToasterTests.tsx

```diff
@@ -29,122 +29,132 @@ describe("OverlayToaster", () => {
     let testsContainerElement: HTMLElement;
     let toaster: Toaster;
 
-    before(() => {
-        testsContainerElement = document.createElement("div");
-        document.documentElement.appendChild(testsContainerElement);
-        toaster = OverlayToaster.create({}, testsContainerElement);
-    });
+    describe("create with default props", () => {
+        before(() => {
+            testsContainerElement = document.createElement("div");
+            document.documentElement.appendChild(testsContainerElement);
+            toaster = OverlayToaster.create({}, testsContainerElement);
+        });
 
-    afterEach(() => {
-        toaster.clear();
-        ReactDOM.unmountComponentAtNode(testsContainerElement);
```

Comment:
We were previously calling `ReactDOM.unmountComponentAtNode` on the wrong DOM element. The `OverlayToaster.create()` method creates a new `<div>` within `testsContainerElement` to render on.